lib/rake/test.rake

```diff
@@ -20,7 +20,7 @@ namespace :test do
   task :regular_ui do
     Dir.chdir(dashboard_dir('test/ui')) do
       ChatClient.log 'Running <b>dashboard</b> UI tests...'
-      failed_browser_count = RakeUtils.system_with_chat_logging 'bundle', 'exec', './runner.rb', '-d', 'test-studio.code.org', '--parallel', '120', '--magic_retry', '--with-status-page', '--fail_fast'
+      failed_browser_count = RakeUtils.system_with_chat_logging 'bundle', 'exec', './runner.rb', '--parallel', '2', '--magic_retry', '--with-status-page', '--fail_fast'
```

Comment:
This is already the default, so there's no reason to specify `-d` on test.

One approach I could have taken was to specify `-d` with CDO.uitest_dashboard_domain here, but that would make handling reruns a pain.